GitTrends.Android/GitTrends.Android.csproj

```diff
@@ -58,7 +58,8 @@
     <AndroidLinkTool>r8</AndroidLinkTool>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'AppStore|AnyCPU' ">
-    <DebugType></DebugType>
+    <DebugType>
+    </DebugType>
```

Comment:
Let's remove these syntax changes in the CSPROJ from the Pull Request.

It doesn't look like anything actually changed. I think Visual Studio likely did an auto-format.

Reply:
Actually, we can remove the whole file from the PR. It has been modified by Visual Studio, I didn't touch it.

I should have taken a look at this before committing sorry...